settings.json

```diff
@@ -1,753 +1,973 @@
+                {
+                    "name": "Brand Danger",
+                    "oldNames": ["Danger"],
+                    "class": "groupbox-danger"
+                },
+                {
+                    "name": "Transparent",
+                    "class": "groupbox-transparent"
+                }
+            ]
+        },
+        {
+            "name": "Callout style",
+            "type": "Toggle",
+            "description": "Enable the groupbox callout functionality to highlight the importance of the groupbox.",
+            "class": "groupbox-callout"
+        }
+    ],
+    "StaticImageViewer": [
+        {
+            "name": "Style",
+            "type": "Dropdown",
+            "description": "Choose the style of your image.",
+            "options": [
+                {
+                    "name": "Rounded",
+                    "class": "img-rounded"
+                },
+                {
+                    "name": "Thumbnail",
+                    "class": "img-thumbnail"
+                },
+                {
+                    "name": "Circle",
+                    "class": "img-circle"
+                }
+            ]
+        },
+        {
+            "name": "Center",
+            "type": "Toggle",
+            "description": "Align the image in the center of an element.",
+            "class": "img-center"
+        },
+        {
+            "name": "Fit",
+            "type": "Dropdown",
+            "description": "Choose the image fit.",
+            "options": [
+                {
+                    "name": "Fill",
+                    "class": "img-fill"
+                },
+                {
+                    "name": "Contain",
+                    "class": "img-contain"
+                },
+                {
+                    "name": "Cover",
+                    "class": "img-cover"
+                },
+                {
+                    "name": "Scale-down",
+                    "class": "img-scale-down"
+                }
+            ]
+        }
+    ],
+    "DynamicImageViewer": [
+        {
+            "name": "Style",
+            "type": "Dropdown",
+            "description": "Choose the style of your image.",
+            "options": [
+                {
+                    "name": "Rounded",
+                    "class": "img-rounded"
+                },
+                {
+                    "name": "Thumbnail",
+                    "class": "img-thumbnail"
+                },
+                {
+                    "name": "Circle",
+                    "class": "img-circle"
+                }
+            ]
+        },
+        {
+            "name": "Center",
+            "type": "Toggle",
+            "description": "Align the image in the center of an element.",
+            "class": "img-center"
+        },
+        {
+            "name": "Fit",
+            "type": "Dropdown",
+            "description": "Choose the image fit.",
+            "options": [
```

Comment:
Introduce new design properties for DynamicImageViewer to help with fitting an image